orion-ui/src/pages/TaskRun.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <p-layout-well class="task-run">
```

Comment:
Pretty much every component on this page has `v-if="task-run"`. Can we put that v-if on the root element so we don't have it repeated?